src/main/java/study/StringCalculator.java

```diff
@@ -0,0 +1,39 @@
+package study;
+
+
+public class StringCalculator {
+
+	public int calculate(String str) {
+
+		if(str == null || str.equals("")) {
```

Comment:
```suggestion
		if( "".equals(str)) {
```
와 같이 변경한다면 조금 더 간결하게 코드를 작성할 수 있을 것 같네요.